UPGRADE.md

```diff
@@ -1,3 +1,38 @@
+## UPGRADE FOR `1.9.x`
+
+### FROM `1.8.x` TO `1.9.x`
+
+- `translations`:
+
+If you are using translations, the default locale provider `sylius.translation_locale_provider.immutable` is using two new translation options.
+
+```yaml
+sylius_resource:
+    translation:
+        enabled_locales: []
+        default_locale: null
+```
+
+Before `1.9.x` it was configured with `%locale%` parameter.
+If you still use this `%locale%`parameter, you can configure translation like this:
+
+```yaml
+sylius_resource:
+    translation:
+        enabled_locales: ['%locale%']
+        default_locale: '%locale%'
+```
+
+If you use `%kernel.default_locale%` and `%kernel.enabled_locales%` parameters, you can configure translation like this:
+
+
+```yaml
+sylius_resource:
+    translation:
+        enabled_locales: '%kernel.enabled_locales%'
+        default_locale: '%kernel.default_locale%'
+```
```

Comment:
````suggestion
New configuration options were introduced to set the default and the enabled locales used by the default translation locale provider (`sylius.translation_locale_provider.immutable` service):

```yaml
sylius_resource:
    translation:
        enabled_locales: []
        default_locale: null
```

Before version `1.9.x` the provider was configured with `locale` parameter.
If those options are not configured, for backward compatibility, it will fallback to the `locale` parameter if it's defined.
This is deprecated and will be removed in version 2.0.
If you want to keep using the `locale` parameter starting with version 2.0, then add the following configuration:

```yaml
sylius_resource:
    translation:
        enabled_locales: ['%locale%']
        default_locale: '%locale%'
```

Starting with version 2.0, the default values will be set from  `kernel.default_locale` and `kernel.enabled_locales` parameters.


```yaml
sylius_resource:
    translation:
        enabled_locales: ['%kernel.enabled_locales%']
        default_locale: '%kernel.default_locale%'
```
````